src/enums/filingNames.ts

```diff
@@ -6,7 +6,7 @@ export enum FilingNames {
   CORRECTION = 'Correction',
   INCORPORATION_APPLICATION = 'Incorporation Application',
   NAME_REQUEST = 'Name Request',
-  NOTICE_OF_ALTERATION = 'Alteration Notice',
+  NOTICE_OF_ALTERATION = 'Changes to Company Information',
```

Comment:
This will affect the filing name in the History List. Did you see a design for this? (I didn't.)

Reply:
According to my story, "Changes to Company Information" is the subtitle.
https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/4575

Reply:
hmm, this is al i have to work off! It does show the same title in the history list as well 🤔 

https://preview.uxpin.com/62ac91b2c4745163586b730ec154e8ea024bd8ed#/pages/133498721/simulate/no-panels

Reply:
🤷‍♂️ 
I have asked for clarity on this but still waiting for an answer.

I like your uxpin but it directly conflicts with my story.

Reply:
oh no! hahah.. classic :) 

No rush on this, this PR can hang out until we get some answers.

Reply:
Mine too.

Should we link in Thor's vid? lol

Reply:
My ticket has a [new uxpin](https://preview.uxpin.com/fac96ab3f3769b5940b0444784eb73a2b2606b0e#/pages/136199952/simulate/no-panels).

The alteration filing title in the Filings History List is different than the alteration filing in the Todo List, so I'd argue you should not change the filing name as above.

I'd also argue that the Todo List should display simply "Alteration Notice" (or "Alteration:") since you don't know whether the filed alteration will be for (1) a legal type change, or (2) some other change to company information (*). But if not then I still think you need to set your title somewhere and not change the filing name.

(*) The Filing History List identifies only these 2 types of changes in an alteration, so I think it makes sense that the Todo List should be similar.

Reply:
I'm not opposed to leaving it. I will revert it back to `Alteration Notice` and we can push it through to UX. 
When they see it all together we can update if they prefer :)

Reply:
1. I think you should not change the filing name just to suit one display item. It impacts other things.
2. Check with BA/UX on what to call the Todo List item. I do not have authority on this.